static/app/components/compactSelect/listBox.tsx

```diff
@@ -256,13 +265,21 @@ function ListBox<Value extends React.Key>({
         });
       }
     },
-  });
+    [
+      focusManager,
+      firstFocusableKey,
+      lastFocusableKey,
+      listState.selectionManager.focusedKey,
+      shouldFocusWrap,
+      listBoxProps,
```

Comment:
Should you really pass the entire object? You probably want to just pass `listBoxProps.onKeyDown` since that's all you're using

Reply:
I get an eslint error if I only add `listBoxProps.onKeyDown` to the deps array:
<img width="872" alt="Screenshot 2023-02-21 at 2 10 42 PM" src="https://user-images.githubusercontent.com/44172267/220470101-be2fe8b2-9f32-4968-9116-efcd49c09189.png">